orchagent/portsorch.cpp

```diff
@@ -3351,6 +3391,26 @@ void PortsOrch::updateDbPortFlapCount(Port& port, sai_port_oper_status_t pstatus
     m_portTable->set(port.m_alias, tuples);
 }
 
+void PortsOrch::updateDbPortOperError(Port& port, PortOperErrorEvent *pevent)
+{
+    SWSS_LOG_ENTER();
+
+    auto key = pevent->getDbKey();
+    vector<FieldValueTuple> tuples;
+    FieldValueTuple tup1("oper_error_status", std::to_string(port.m_oper_error_status));
```

Comment:
Please check the latest change to notification structure - https://github.com/opencomputeproject/SAI/pull/2087
Was discussed in the SAI community meeting

Reply:
@eddyk-nvidia that is for future use case. I can't wait for that to merge and start using as its already not meeting 202411 release time.